src/Umbraco.Web.UI/config/imageprocessor/processing.config

```diff
@@ -14,6 +14,7 @@
     <plugin name="Filter" type="ImageProcessor.Web.Processors.Filter, ImageProcessor.Web" />
     <plugin name="Flip" type="ImageProcessor.Web.Processors.Flip, ImageProcessor.Web" />
     <plugin name="Format" type="ImageProcessor.Web.Processors.Format, ImageProcessor.Web" enabled="true" />
+    <plugin name="Gamma" type="ImageProcessor.Web.Processors.Gamma, ImageProcessor.Web" />
```

Comment:
Do we need to add a config transform to add this for upgrades or is this optional?

Reply:
Nope, this was added by the ImageProcessor NuGet so we don't have to worry about that.